.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,14 @@
+name: auto-merge
+on:
+  pull_request:
+    branches-ignore:
+      - 'gh-pages'
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
+        with:
+          target: minor
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I'm pretty sure this token does not have enough permissions to merge https://github.com/marketplace/actions/dependabot-auto-merge#token-scope

you'd need to make a personal access token, which would then be like you merging them all manually. do you want that responsibility? :)

Reply:
Ah, I see, that’s the same issue as `semantic-release`, then… Could we use a @commercial-dev token?

Reply:
I'm pretty sure they're per-user, it's an interesting question though...